src/Phinx/Db/Adapter/PostgresAdapter.php

```diff
@@ -747,6 +748,7 @@ public function getPhinxType($sqlType)
                 return static::PHINX_TYPE_CHAR;
             case 'text':
             case 'json':
+            case 'jsonb':
```

Comment:
I was thinking so myself, but I just followed what was being done with the json type before. I'll update my branch and see what the CI tests show.